.drone.yml

```diff
@@ -71,24 +71,26 @@ pipeline:
       - codecov_token
     commands:
       - export BUILD_NUMBER=${DRONE_BUILD_NUMBER}
-      - make all
+      - make appliance components vic-machine
```

Comment:
This should still `make all` - that way we run the linting/tool checks before building.

Reply:
Yes. It 's temporary change for quick execution on integration tests.